logger/mask.go

```diff
@@ -229,3 +246,27 @@ func willMask(valueToReplace string, restrictionType MaskRestrictionType) bool {
 
 	return true
 }
+
+// MaskFormURLEncoded mask parts of the key paths values from the input form encoded string with replacement
+func MaskFormURLEncoded(form string, maskChar string, toMasks []MaskData) string {
```

Comment:
uhm do you think it is a hassle to mask the form without marshalling and unmarshalling to-from json?
```
formData url.Values // instead of string
for key, toMask := range formData {
  formData.Add(key, someMaskFunc(toMask))    
}
```

Reply:
Done in 3bb0f5a.

Wrote benchmark test - e5b318a. Committed benchmark performance of the old way vs the new way - d960dfc  - so that we can check the history of the benchmark performance before and after changing (utilizing git commits).